requirements.txt

```diff
@@ -1,4 +1,5 @@
 # note there are also rpm packaged deps that aren't
 # setup in pypi or with setup.py
 ../python-rhsm
+decorator
```

Comment:
Ack. Do you think we should do this for all requirements from the spec then, for completeness? But I'm good with your explanation; for the scope of this PR, I'm good with this change to `requirements.txt` as-is.